### DISCUSSION TITLE
Procedural question: how to trigger a team review of a languishing PR

### DISCUSSION THREAD

**Discussion by chauser** [wpilibsuite/2025Beta#76] [Q&A] [answered]
A PR I submitted in April has not been reviewed. I think it is ready and has completed its checks. I suspect that for some reason the automated systems have not brought it to the attention of those who would review it. I'm guessing this message will get it unstuck, but in general what is the preferred way to approach this -- I don't want to be a pest, but I would like to get it reviewed (and fixed need be) so it can progress toward completion. https://github.com/wpilibsuite/allwpilib/pull/6490

**Comment by KangarooKoala** [comment]
I'm not a member of the wpilibsuite team, so I cannot speak absolutely on their behalf. However, I believe that there are automated systems for notifying wpilibsuite members of new PRs, but there are none to bring old PRs to their attention again. It's unfortunate that sometimes PRs aren't reviewed for a while (I had one that wasn't reviewed for months), but I think the reason PR authors are responsible for reminding reviewers of old PRs is that the volunteer team of reviewers (who have their own full time jobs and own contributions to work on) is limited in how much time they can spend on PRs that the authors might have abandoned anyways.

I think the preferred way to bring PRs to the attention of reviewers is to either add a comment to the PR (which should give them another notification) or bring up the PR in the #programming-discussion or #programming-quick-help channels of the [FRC server on Discord](https://discord.gg/frc). You can also use the GitHub review requesting system, but that's a bit limited.

**Comment by rzblue** [comment, marked as answer]
Apologies for letting that PR go stale- we try our best to stay on top of PRs but sometimes they slip through under the radar. If you ever feel that a PR was forgotten, feel free to ping a relevant member or group of the WPILib team in a comment - suggestions are usually linked at the top of the PR. Some of us are also active on the FRC discord and respond to pings there.

**Comment by chauser** [comment]
Thanks!
